.github/workflows/examine-python-bindings.yml

```diff
@@ -183,32 +183,3 @@ jobs:
           args: --template .license_template
 
       # TODO: Check the Python code too
-
```

Comment:
Can't comment there, but if all bindings are in the same workspace and use the same target folder, we can also use the same cache for all? Since we're limited there this might work for all better at the end?

Reply:
Very much possible but I'd do it in a subsequent PR as I want to reorganise everything